src/app/(main)/wines/[wineid]/page.tsx

```
@@ -3,8 +3,8 @@ import ReviewForm from '@/feature/reviews/components/review-form/ReviewFormButto
 import WineProgressChart from '@/feature/wines/components/wine-progress';
 import WineDetailCardSection from '@/feature/wines/detail/components/WineDetailCardSection';
 import WineDetailReviewList from '@/feature/wines/detail/components/WineDetailReviewList';
-import WineDetailTitle from '@/feature/wines/detail/components/WineDetailTitle';
 import WineFlavorProfileSection from '@/feature/wines/detail/components/WineFlavorProfileSection';
+import Error from '@/shared/components/common/error';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)